packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -72,21 +72,9 @@ export const getPatientByDemo = async ({
 
     // Basic Implementation with semi-arbitrary threshold
     const jw_score = calculatePatientSimilarity(patient.data, demo);
-    if (jw_score >= 0.95) {
+    if (jw_score >= JW_THRESHOLD) {
       return patient;
     }
```

Comment:
We could encapsulate this a bit further, make this so we ask whether they are the same and let the code on the other file/function/class do its thing without us on this file having to know there's a score and what the threshold is?

Like:

```ts
if (isMatchingDemographics(patient.data, demo)) return true;
return false;
```